### PR TITLE
add sponsorship line item creation (#86)

### PR DESCRIPTION
**Pull request by milesmahon**
Add [support for sponsorship line items](https://github.com/prebid/line-item-manager/issues/86).

### CODE REVIEW

File: tests/resources/cfg.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ order:
 # Line Items (required)
 # Allowed Templating Key Words: Bidder, Media, CPM, Misc
 #
-# Supported Types: price_priority, standard
+# Supported Types: price_priority, standard, sponsorship
```
Comment (dshore) [inline tests/resources/cfg.yml:72]: This should be added here as well:
https://github.com/prebid/line-item-manager/blob/master/line_item_manager/conf.d/line_item_manager.yml#L93

File: tests/resources/cfg.yml [hunk 2]
```diff
@@ -78,6 +78,10 @@ line_item:
   name: "Prebid-{{ bidder_name }}-{{ media_type }}-{{ time }} @ {{ cpm }}"
   item_type: "price_priority"
   # Optional
+  # goal: # required if sponsorship item_type
+    # units: 5
+    # unitType: "VIEWABLE_IMPRESSIONS"
+    # goalType: "LIFETIME"
```
Comment (dshore) [inline tests/resources/cfg.yml:84]: This should be added here as well:
https://github.com/prebid/line-item-manager/blob/master/line_item_manager/conf.d/line_item_manager.yml#L103

**Review by dshore — CHANGES_REQUESTED** [review]
If you bump this to '1' we can cut a dev release:
https://github.com/prebid/line-item-manager/blob/master/line_item_manager/__init__.py#L8
I like to use it for a few days at least before doing a tagged release, and publish to pypi.

**Comment by milesmahon** [comment]
done with 3a9ca6c

**Review by dshore — APPROVED** [review]

**Comment by dshore** [comment]
@milesmahon this is available as a dev release:
```
$ pip uninstall line-item-manager
$ pip install https://github.com/prebid/line-item-manager/archive/master.zip
$ line_item_manager --version
line-item-manager version 0.2.4-dev1
```

**Pull request merged**